languages/jelly/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:buster
+
+WORKDIR /tmp
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y \
+        wget \
+        python3 \
+        python-pip3 \
+        && \
+    wget https://github.com/DennisMitchell/jellylanguage/archive/master.tar.gz && \
+    pip3 install --upgrade https://github.com/DennisMitchell/jellylanguage/archive/master.tar.gz
```

Comment:
`--upgrade` is not necessary here (because the Docker file is built from scratch), and may even cause problems with package conflicts (I'm not sure how `pip` would handle this when installing only from a file, but I think it's cleaner without it anyway)